lib/findUndefinedIdentifiers.js

```diff
@@ -0,0 +1,38 @@
+function iter(node, definedInScope = ['module']) {
+  const result = [];
+  if (typeof node !== 'object') {
+    return result;
+  }
+  Object.keys(node).forEach((key) => {
```

Comment:
Node 4 doesn't support it, so I ended up having to revert back to `Object.keys`.